server/server.go

```diff
@@ -525,6 +525,12 @@ func (server *Server) prepareServer(entryPointName string, router *middlewares.H
 		return nil, err
 	}
 
+	middlewaresStr := ""
+	for _, handler := range negroni.Handlers() {
+		middlewaresStr += reflect.TypeOf(handler).String() + " "
```

Comment:
Ok :) To be honest I am not sure how useful I find this. With a simple configuration I currently get the following output:
```
Middlewares: negroni.HandlerFunc *middlewares.Logger *stats.Stats negroni.HandlerFunc
```
Additionally I have for example retries configured. It is not mentioned here, as this middleware is setup in the `loadConfig()` method and specific to a backend. Therefore we should maybe make this message more explicit or extend the logging also on other positions. WDYT?